include/psp2/net/http.h

```diff
@@ -1,6 +1,6 @@
 /**
  * \usergroup{SceHttp}
- * \usage{psp2/net/http.h,SceHttp_stub}
+ * \usage{psp2/net/http.h,SceHttp_stub,SCE_SYSMODULE_HTTP}
```

Comment:
Add a warning about loading `SCE_SYSMODULE_HTTPS` on the function `sceHttpsLoadCert()`.